src/component.js

```diff
@@ -13,6 +13,7 @@ import { Fragment } from './create-element';
 export function Component(props, context) {
 	this.props = props;
 	this.context = context;
+	this.__data = {};
```

Comment:
Todo: find a way to only do this in one spot, ideally with defaults for the stuff we use